pom.xml

```diff
@@ -21,7 +21,7 @@
         <repository>
             <id>jboss-releases-repository</id>
             <name>JBoss Releases Repository</name>
-            <url>https://repository.jboss.org/nexus/service/local/staging/deploy/maven2/</url>
+            <url>https://repository.jboss.org/nexus/content/repositories/releases/</url>
```

Comment:
What's the reason for this?

Reply:
The `https://repository.jboss.org/nexus/service/local/staging/deploy/maven2` is our internal staging repo and needs jboss account to access:

![image](https://user-images.githubusercontent.com/201907/59236165-75d4bc00-8c27-11e9-83db-7768ee3f73a4.png)

The `https://repository.jboss.org/nexus/content/repositories/releases` is public:

![image](https://user-images.githubusercontent.com/201907/59236172-8127e780-8c27-11e9-8b48-94ae3e043800.png)

Should we use the public one?

Reply:
In addition, we can remove the repository settings to use the released resteasy releases. wdyt?